fs.go

```diff
@@ -52,6 +52,7 @@ const (
 type Filesystem interface {
 	Basic
 	TempFile
+	TempDir
```

Comment:
Adding TempDir to the interface will force a major upgrade, I fill that doesn't worth to do it.
Maybe in the future with more updates.